fcomm/src/snapshots/fcomm__test__snapshot_serialized_expressions-4.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: fcomm/src/lib.rs
+expression: req
+---
+{
+  "commitment": "000000000000000000000000000000000000000000000000000000000000007b",
```

Comment:
Observation: this doesn't look like a commitment but rather like a small constant. That's what initially tipped me off that these examples are not 'realistic'.

Reply:
That is correct.